### PR TITLE
Consistency fixes: "reporting period" to "reference period"

### PR DESCRIPTION
**Pull request by schuur**
Change mentions of "reporting period" to "reference period" in descriptions, in line with parameter and attribute renaming from reportingPeriod to referencePeriod.

### CODE REVIEW

File: spec/v2/index.bs [hunk 1]
```diff
@@ -191,7 +191,7 @@ below). See [[#api-auth]] for details.
 </figure>
 
 1. The data recipient sends a `ProductFootprintFragment`  to the `Events Endpoint` of the data owner. The data recipient should always include 1 or more product ids in property `productIds`. The data recipient should limit each fragment to exactly 1 specific product (e.g. a specific type of apple instead of all apples that somebody is offering)
-    1. If the recipient is requesting a specific reporting period, it should include the reporting
+    1. If the recipient is requesting a specific reference period, it should include the reporting
```
Comment (bhadley) [inline spec/v2/index.bs:194]: Also the "reporting period" mentioned after the comma should change to "reference period". I counted a total of 11 references to "reporting period" in the tech specs, and this is the one we are missing in this ADR.

**Review by bhadley — CHANGES_REQUESTED** [review]
Should we also perhaps add a small comment to the definition of the properties referencePeriodStart and referencePeriodEnd, to indicate these are also known as "reporting period"? Just because I anticipate this question - some industry standards use "reporting period" instead of "reference period", so it could cause confusion without clarification.

File: spec/v2/index.bs (hunk 1 above)
Comment (schuur) [reply spec/v2/index.bs:194]: Sharp! This one should be changed as well of course

**Review by bhadley — APPROVED** [review]
LGTM!

**Pull request merged**